book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Your method looks great 👏 In addition to what you have done, I can see that you pass `rental` object as an argument but also remember that in order to create a rental object, you have to provide pass in a book object. That I why I think that assigning rental a book attribute (on line 12) should come before pushing the rental object into the @rentals array (on line 11) What do you think? 
However, the best way I would suggest that you do it is to make this method accept date and person parameters and then use them to create a rental object and then push it into @rentals array all done within this function. Like this 👇 

```ruby
  def add_rental(person, date)
    rental = Rental.new(date, self, person)
   @rentals.push(rental)
  end
```